dragonfly/actions/action_base.py

```diff
@@ -259,7 +259,7 @@ class Repeat(object):
 
         Integer Repeat factors ignore any supply data::
 
-            >>> integer = Repeat(3)
+            >>> integer = Repeat(count=3)
```

Comment:
Should we maybe add a note here about the `Action() * 3` shortcut?

Reply:
Sure, I forgot you could do that. It is documented in the doc tests for actions. We could probably have `Action() * "n"` as well.

Reply:
I've added a simple example demonstrating this.

Reply:
Nice, that should make it easily discoverable.

Reply:
Yep.